index.html

```diff
@@ -2,98 +2,226 @@
+                    <ul class="canopy">
+                        <li>CANOPY</li>
+                        <li class="bullets"><span class="dot"></span></li>
+                        <li class="o-li">Back End Dev</li>
+                        <li class="bullets"><span class="dot"></span></li>
+                        <li class="o-li">2015</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
+                <ul class="categories hex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <div class="see-project">
+                    <a href="#"><button type="button" class="hex">See Project</button></a>
+                </div>
+            </div>
+        </section>
+    </div>
+    <!-- About myself -->
+    <section class="about-myself">
+        <h2 class="about">About Myself</h2>
+        <p class="text">
+            Hello I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h2 class="hex connect">LET’S CONNECT</h2>
+        <ul class="social-media">
+            <li clalss="socials">
+                <a href="#"><img src="images/socials/twitter.svg" /></a>
+            </li>
+            <li clalss="socials">
+                <a href="#"><img src="images/socials/Linkedin.svg" /></a>
+            </li>
+            <li clalss="socials">
+                <a href="#"><img src="images/socials/m.svg" /></a>
+            </li>
+            <li clalss="socials">
+                <a href="#"><img src="images/socials/Vector1.svg" /></a>
+            </li>
+            <li clalss="socials">
+                <a href="#"><img src="images/socials/Vector2.svg" /></a>
+            </li>
+        </ul>
+        <div class="see-project">
+            <a href="#"><button type="button" class="hex">Get my resume</button></a>
+        </div>
+
+        <div class="languages-list">
+            <ul>
+                <li class="lang-title">
+                    <p class="langusge">Languages</p>
+                    <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
+                </li>
+                <li class="lang">
+                    <img src="images/langusges/js.png" alt="js" />
+                    <p>JavaSript</p>
+                </li>
+                <li class="lang">
+                    <img src="images/langusges/html.png" alt="js" />
+                    <p>HTML</p>
+                </li>
+                <li class="lang">
+                    <img src="images/langusges/css.png" alt="js" />
+                    <p>CSS</p>
+                </li>
```

Comment:
Kindly ensure that your design looks exactly as the actual figma design. As shown below, there is suppose to be some margins between the languages, but it is not visible in your design since they look cramped together. Kindly check and update it accordingly. :+1: 

### Your design 

![Screenshot from 2023-02-18 22-11-58](https://user-images.githubusercontent.com/65233860/219901749-9e2f4596-ab7f-4fdc-b952-359bc55ce8e8.png)


### Figma design

![Screenshot from 2023-02-18 22-12-37](https://user-images.githubusercontent.com/65233860/219901750-7e6b4f2a-97a9-4f6f-96a3-98edb9563042.png)